edit/edit.go

```diff
@@ -41,17 +41,17 @@ var (
 // InterpretLabelForWorkspaceLocation returns the name of the BUILD file to
 // edit, the full package name, and the rule. It takes a workspace-rooted
 // directory to use.
-func InterpretLabelForWorkspaceLocation(root string, target string) (buildFile string, pkg string, rule string) {
+func InterpretLabelForWorkspaceLocation(root string, target string) (buildFile string, repo string, pkg string, rule string) {
```

Comment:
nit: while you are here can group inputs, outputs

(root, target string) (buildFile, repo, pkg, rule string)

Reply:
I updated the change.  Thanks for reviewing.